### PR TITLE
release-v1.1: update release references 

### PR DESCRIPTION
**Pull request by schristoff**
Signed-off-by: Sarah Christoff <sarah.christoff13@gmail.com>

### CODE REVIEW

File: content/docs/api_reference/config/v1alpha1.md [hunk 1]
```diff
@@ -1,29 +1,23 @@
----
-title: "Config v1alpha1 API Reference"
-description: "Config v1alpha1 API reference documentation."
-type: docs
----
-
```
Comment (nojnhuh) [inline content/docs/api_reference/config/v1alpha1.md:6]: We should keep these headers.

File: config.toml [hunk 2]
```diff
@@ -82,13 +82,16 @@ prism_syntax_highlighting = true
 project_home = "https://openservicemesh.io/"
 
 # Version params
-osm_branch = "main"
```
Comment (nojnhuh) [inline config.toml:85]: Did you branch off of `main`? This PR should be branched off of `release-v1.1`. I think that will show a lot less has actually changed.

File: netlify.toml [hunk 3]
```diff
@@ -10,5 +10,5 @@
 
 [[redirects]]
   from = "https://docs.openservicemesh.io/*"
-  to = "https://release-v1-0.docs.openservicemesh.io/:splat"
+  to = "https://release-v1-1.docs.openservicemesh.io/:splat"
```
Comment (nojnhuh) [inline netlify.toml:13]: This change won't do anything on the release branch I don't think. I think this only has any effect when done on `main`.

File: content/docs/api_reference/config/v1alpha1.md [hunk 4]
```diff
@@ -1070,5 +1064,5 @@ for all inbound and ingress traffic in the mesh.</p>
 <hr/>
 <p><em>
 Generated with <code>gen-crd-api-reference-docs</code>
-on git commit <code>386d92a1</code>.
+on git commit <code>11d0c2df</code>.
```
Comment (nojnhuh) [inline content/docs/api_reference/config/v1alpha1.md:1067]: Usually I'd say let's wait to generate the docs until the v1.1.0 tag is created on the OSM repo so then this hash would match the tag's commit. These docs aren't going to change though before then so practically it doesn't matter.

**Review by nojnhuh — CHANGES_REQUESTED** [review]

File: netlify.toml (hunk 3 above)
Comment (schristoff) [reply netlify.toml:13]: Should I delete and make new PR for main?

File: content/docs/api_reference/_index.md [hunk 5]
```diff
@@ -1,5 +0,0 @@
----
```
Comment (nojnhuh) [inline content/docs/api_reference/_index.md:1]: We should keep this.

File: content/docs/api_reference/policy/v1alpha1.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "Policy v1alpha1 API Reference"
```
Comment (nojnhuh) [inline content/docs/api_reference/policy/v1alpha1.md:2]: This should stay the same

File: netlify.toml (hunk 3 above)
Comment (jaellio) [reply netlify.toml:13]: You can make the change in main once this PR is complete and you have created `https://release-v1-1.docs.openservicemesh.io`. Bridget can help with that once you are ready.

Comment (schristoff) [reply netlify.toml:13]: I **think** I did that already #349  - should I do it again?

Comment (jaellio) [reply netlify.toml:13]: Awesome, nope! Didn't realize you had already done that. This change is fine to make on the release branch, but it must also be made on main for `https://docs.openservicemesh.io/` to redirect to `https://release-v1-1.docs.openservicemesh.io`

Comment (schristoff) [reply netlify.toml:13]: #355  exists now 🙏🏻

File: content/docs/guides/install.md [hunk 7]
```diff
@@ -16,6 +16,8 @@ OSM can be run on Kubernetes versions that are supported at the time of the OSM
 
 | Open Service Mesh | Kubernetes  |
 | ----------------- | ----------- |
+| 1.1               | 1.19 - 1.22 |
```
Comment (nojnhuh) [inline content/docs/guides/install.md:19]:
```suggestion
| 1.1               | 1.20 - 1.23 |
```

**Review by nojnhuh — APPROVED** [review]
LGTM. Could you please sign off or squash your commits to please the DCO bot?

**Pull request merged**